ROMFS/px4fmu_common/init.d/airframes/4400_ssrc_fog_x

```diff
@@ -14,7 +14,8 @@
 . /etc/init.d/rc.mc_defaults
 
 # Default rates
-param set-default IMU_GYRO_CUTOFF 60
+param set-default IMU_GYRO_CUTOFF 30
+param set-default IMU_GYRO_NF0 75
```

Comment:
Can't find this param. Did you mean notch filter freq or bandwidth params IMU_GYRO_NF0_FRQ or IMU_GYRO_NF0_BW ?

Reply:
IMU_GYRO_NF0_FRQ, BW should be fine at the default value.